src/features/changePassword.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Change Password
+
+  @password
+  Scenario Outline: Changing password 
+  When user logs into Isanteplus application and goes to the Home page
+  And user clicks System Administration  
+  And User clicks Manage Accounts App and goes to Manage Accounts page
+  And User clicks edit button action and goes to account page
+  And User clicks on modifier under user account detaials and enters username "<userName>" and privillage "<newPrivillage>"
```

Comment:
Hello @mozzy11  @gitcliff , would you look through this  worklow again, this gherkin naming is lacking, this would end with Then syntax by default